app/controllers/topics_controller.rb

```diff
@@ -1314,16 +1314,21 @@ def track_visit_to_topic
       TopicsController.defer_add_incoming_link(hash)
     end
 
-    TopicsController.defer_track_visit(topic_id, ip, user_id, track_visit)
+    TopicsController.defer_track_visit(topic_id, user_id, track_visit)
```

Comment:
Intentionally leaving this and `defer_add_incoming_link` in the same place as it was previously, I don't think it makes sense to move this to the request tracker.